backend/routes/spotify_routes.js

```diff
@@ -137,4 +140,87 @@ router.get('/search', async (req, res) => {
   }
 });
 
+// Playback Controls
+router.post('/play', async (req, res) => {
+  const accessToken = process.env.ACCESSTOKEN;
+
+  if (!accessToken) {
+    return res.status(401).send('Access token not available. Please log in.');
```

Comment:
basically, let's try to standardize all the responses coming from the backend I think the format that ryan came up with is the is pretty good, where every response includes a status code along with at least 2 fields: a boolean `success` and a string `message`, for example:

        `return res.status(404).send({ success: false, message: "Could not retrieve posts. Please try again later." });`

some responses might need to return data as well, such as 

        `return res.status(200).send({success: true, message: "Posts have been retrieved successfully", data: posts});`

data can also be an object

`data: {user: userData, posts: posts}}`

ideally, the data field would always be called `data` regardless of what the data is

And if the return statement is in a `catch` statement, there should be an additional field `error` such as in 

         `return res.status(500).send({ success: false, message: "internal server error", error: error.message});`
         
Also, can we change all the `.json()` statements to `.send()`